cmd/meroxa/turbine/utils.go

```diff
@@ -241,6 +253,20 @@ func trimNonNpmErrorLines(output string) string {
 	return strings.Join(errorLines, "\n")
 }
 
+// TODO: remove this and refactor error handling from turbine-core grpc requests
+// This is needed temporarily to provide a more actionable error when the app has no sources defined.
+// Longer-term, we need to have more specific, actionable errors rather than grpc validation
+// errors surfaced to CLI output.
+func clarifyGrpcErrors(errLog string) string {
+	switch {
+	case strings.Contains(errLog, grpcFuncCollectionErr):
+		return missingSourceCollectionErr
```

Comment:
Is it intended that the `missingSourceCollectionErr` is returned in both `switch` cases here?

Reply:
yeah, this is the intention - the two cases are for whether the error surfaces within the configuration of a function or a destination resource in the turbine app. But the underlying cause/fix is the same - so we return the same error

Reply:
Got it and thank you for clarifying how this error is surfaced!